README.md

```diff
@@ -1,8 +1,87 @@
+
+```
+
+As a simple example, the below command will harmonize the white matter region in a group of images using ComBat:
+```
+$ habet/habet/main.py harmonize --mask_path <white_matter_mask_path> --site_colname <site_colname> -o <harmonized_dir> --harmonization_methods Combat --df_path <df_path>
+```
+
+HABET also features a simple API for adding custom harmonization techniques. This API is still in its early stages, but at a high level:
+1. Add a python file to the `habet/harmonization` directory, giving it the name of the harmonization method you'd like to add
+2. In this file, implement a python class that inherits from `HarmonizationMethod` and implements the `_harmonize()` method (see `combat.py` or `global_scaling.py` as examples).
+
+## Report Generation
+The report generation of HABET summarizes the detected site-effects. The help string for the report generation subcommand is given below.
+
+```
+usage: main.py generate_report [-h] [--mask_path MASK_PATH] --site_colname SITE_COLNAME -o OUTPUT_DIR --im_path_to_site_id_df_path IM_PATH_TO_SITE_ID_DF_PATH [--anova_alpha ANOVA_ALPHA]
+                               [--t_test_alpha T_TEST_ALPHA] [--mtc] [--save_dfs]
+
+optional arguments:
+  -h, --help            show this help message and exit
+  --mask_path MASK_PATH
+                        Path to harmonization mask. Harmonization will only be applied to voxels with intensity value == 1.
+  --site_colname SITE_COLNAME
+                        colname of the site variable in the dataframe at df_path
+  -o OUTPUT_DIR, --output_dir OUTPUT_DIR
+                        Root directory of the report files.
+  --im_path_to_site_id_df_path IM_PATH_TO_SITE_ID_DF_PATH
+                        Path to the dataframe matching each image filenameto the corresponding site id
+  --anova_alpha ANOVA_ALPHA
+                        Significance level for the anova to be done at each voxel
+  --t_test_alpha T_TEST_ALPHA
+                        Significance level for the t-tests to be done at each voxel
+  --mtc                 Conduct Bonferroni multiple testing correction for the anovas and t-tests
+  --save_dfs            Save the anova and t-test dataframes as part of the report Note that these can get large in file size
+
+```
+
+For example, to generate a report for the images harmonized using the command above, you could use the command below:
+```
+$ habet/habet/main.py generate_report --mask_path <white_matter_mask_path> --site_colname <site_colname> -o <report_dir> --im_path_to_site_id_df_path <harmonized_dir>/Combat/image_to_site.csv
+```
+## References
+HABET was first introduced in the following paper, where it was used to harmonize fractional anisotropy (FA) images from the Adult Brain and Cognitive Development (ABCD) study.
+
+> T. Osika, E. Ebrahim, M. Styner, M. Niethammer, T. Sawyer, A. Enquobahrie, ‘Harmonization Benchmarking Tool for Neuroimaging Datasets’. arXiv, 2022.
```

Comment:
Add also, below or above the citation, a link like
```markdown
(arXiv:2211.07869)[https://arxiv.org/abs/2211.07869]
```

Reply:
I'm assuming you mean like 76935778a005a7aaf67e350144ce834d574fbae5? I assumed the () and [] usage was backwards but if you want it the original way, I can switch.

Reply:
Yes exactly! Thanks for fixing I always get that backwards :roll_eyes: 

Ready to merge I think :slightly_smiling_face: